src/customFunctions/customFetch.ts

```diff
@@ -0,0 +1,18 @@
+import { bearerTokenAuthorization } from "./../configs/bearerTokenAuthorization";
+
+export const customFetch = async (
+  input: RequestInfo,
+  init: RequestInit = {}
+) => {
+  init.headers = bearerTokenAuthorization.headers;
+  const response = await fetch(input, init);
+
+  if (!response.ok) {
+    throw new Error(`Response status code: ${response.status}`);
+  }
+
+  return {
+    fullResponse: response.clone(),
```

Comment:
`.clone()` pecka, jen moc lidí asi nepochopí, proč to tam je, tohle si podle mě zaslouží nějakou dokumentaci, protože to asi nikdo neví jen tak